src/appleseed/foundation/meta/tests/test_sss.cpp

```diff
@@ -47,19 +49,58 @@ TEST_SUITE(Foundation_Math_SSS)
         const double c1 = fresnel_moment_c1(eta);
         const double c2 = fresnel_moment_c2(eta);
 
-        const double alpha_prime =
-            compute_alpha_prime(rd, c1, c2);
-
+        const double alpha_prime = compute_alpha_prime(rd, c1, c2);
         return compute_rd(alpha_prime, 2.0 * c1, 3.0 * c2);
     }
 
     TEST_CASE(Rd_AlphaPrime_Roundtrip)
     {
-        EXPECT_FEQ_EPS(0.1, rd_alpha_prime_roundtrip(0.1, 1.3), 0.0001);
-        EXPECT_FEQ_EPS(0.2, rd_alpha_prime_roundtrip(0.2, 1.2), 0.0001);
-        EXPECT_FEQ_EPS(0.4, rd_alpha_prime_roundtrip(0.4, 1.3), 0.0001);
-        EXPECT_FEQ_EPS(0.6, rd_alpha_prime_roundtrip(0.6, 1.4), 0.0001);
-        EXPECT_FEQ_EPS(0.8, rd_alpha_prime_roundtrip(0.8, 1.3), 0.0001);
-        EXPECT_FEQ_EPS(1.0, rd_alpha_prime_roundtrip(1.0, 1.5), 0.0001);
+        EXPECT_FEQ_EPS(0.1, rd_alpha_prime_roundtrip(0.1, 1.3), 0.001);
+        EXPECT_FEQ_EPS(0.2, rd_alpha_prime_roundtrip(0.2, 1.2), 0.001);
+        EXPECT_FEQ_EPS(0.4, rd_alpha_prime_roundtrip(0.4, 1.3), 0.001);
+        EXPECT_FEQ_EPS(0.6, rd_alpha_prime_roundtrip(0.6, 1.4), 0.001);
+        EXPECT_FEQ_EPS(0.8, rd_alpha_prime_roundtrip(0.8, 1.3), 0.001);
+        EXPECT_FEQ_EPS(1.0, rd_alpha_prime_roundtrip(1.0, 1.5), 0.001);
+    }
+
+
+    TEST_CASE(NormalizedDiffusionCdfPdf)
+    {
+        const size_t max_iters = 50;
```

Comment:
Constants are in CamelCase. Also, rename to IterationCount since it's not really a maximum number of iterations, just an actual number of iterations.